Issue1 create home page

shashankjndl20 opened the pull request:
Please First check the functionality and code
 I think Create homepage part is completed

shashankjndl20 committed:
Update index.html

1. Create two buttons - All posts and Create post
2.  All posts directed to bloglist.html 
3. Create post modal

shashankjndl20 committed:
Homepage Commit

1.Function for Create post modal
2. add event listener for all posts

shashankjndl20 committed:
Homepage Commit

shashankjndl20 committed:
Homepage Commit

 parent and input.postButton for createpost and all post buttons

shashankjndl20 committed:
Homepage Commit

parimalvfx commented:
Issue #1

parimalvfx commented on index.html:
```diff
@@ -18,31 +18,94 @@
 
-        	<!-- Insert code for All Posts & Create Post button here -->
+            <!-- Insert code for All Posts & Create Post button here -->
+            <div id ="parent">
```
We don't need `<div id ="parent">`, existing `<div class="postButtons">` will serve the purpose.

shashankjndl20 replied:
yes issue resolved

parimalvfx commented on index.html:
```diff
@@ -18,31 +18,94 @@
     	<!-- Insert code for Sign Up modal here -->
-
+            <div class="modal" id="signUpModal">
+            <div class="modalContent">
```
Please make sure indentation is correct.

shashankjndl20 replied:
ok

parimalvfx commented on js/index.js:
```diff
@@ -0,0 +1,17 @@
+function toggleModal1(modalID) {
```
We have already wrote `toggleModal` function in _header.css_, do we need this duplicated code?

shashankjndl20 replied:
removed from index.js toggleModal1

parimalvfx commented on css/index.css:
```diff
@@ -11,4 +11,98 @@ body{
+
+
+
+form {
```
Defined in _header.css_, can be removed.

shashankjndl20 replied:
removed from index.css

parimalvfx commented on css/index.css:
```diff
@@ -11,4 +11,98 @@ body{
+    justify-content: center;
+    font-weight: bolder;
+}
+label {
```
Defined in _header.css_, can be removed.

parimalvfx commented on css/index.css:
```diff
@@ -11,4 +11,98 @@ body{
+    background-color: #F1F1F1;
+    
+}
+h1 {
```
Defined in _header.css_, can be removed.

shashankjndl20 replied:
removed from index.css

parimalvfx commented on css/index.css:
```diff
@@ -11,4 +11,98 @@ body{
+margin-top : 10px;
+  
+}
+.modal1 {
```
`modal`, `modalContent` and `modalClose` is defined in _header.css_, can be removed.

shashankjndl20 replied:
removed from index.css
.modalContent1 cannot be removed because of width: 70 % create post requirement

shashankjndl20 committed:
Merge branch 'master' into issue1_create_home_page

parimalvfx commented on css/index.css:
```diff
@@ -11,4 +11,98 @@ body{
 
 /* Set a style for the rest of the elements besides the header elements on the index.html page */
-                                 
+#parent {
```
Please use `.postButtons` instead of `#parent`.

shashankjndl20 replied:
trying my best but when i removing parent id and merging the in .postbuttons then buttons are not going in center.

parimalvfx commented on index.html:
```diff
@@ -18,31 +18,94 @@
-        	<!-- Insert code for All Posts & Create Post button here -->
+            <!-- Insert code for All Posts & Create Post button here -->
+            <div id ="parent">
+                 <input class="postButton" type="button" id="allposts" value="All Posts">
```
Is there any use case for having `class="postButton"` to input buttons?

shashankjndl20 replied:
we can use id also

parimalvfx replied:
Class makes sense, let's go with it.

shashankjndl20 committed:
HomePageUpdated


.modalContent1{
width : 70 %       - Due to the larger modal of create post so modalContent1 cannot be removed 
}

shashankjndl20 committed:
HomepageUpdated

parimal - There is a misplace of code. You have inserted sigin modal code in create post modal place.

shashankjndl20 committed:
HomepageUpdated

shashankjndl20 committed:
HomepageUpdated

shashankjndl20 committed:
Homepage Issues Resolved

shashankjndl20 committed:
Homepage Issues Resolved
